webpack/scenes/Subscriptions/Manifest/ToggleManifestSCA.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch } from '@patternfly/react-core';
```

Comment:
I think this will be our first PF4 component on a PF3 page.  Nice!  I'm on board, but want to double check with @johnpmitsch - do you foresee any issues here?

Reply:
as long as it doesn't look too out of places, I don't see an issue with it :+1: